clerk/middleware.go

```diff
@@ -30,3 +31,21 @@ func addSessionToContext(request *http.Request, session *Session) *http.Request
 	updated := context.WithValue(ctx, ActiveSession, session)
 	return request.WithContext(updated)
 }
+
+func WithNetworkless(client Client) func(handler http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			token := r.Header.Get("Authorization")
+
```

Comment:
❓ I think it'll be better DX to check here if the header is empty and if it is, return a respective error indicating the issue.

Reply:
I agree with @agis about the extra-check about the token being empty. But I would choose to place this check into the `VerifyToken` function since we already have a number of checks there regarding the token and the `VerifyToken` can also be used directly (without having to go via the middleware).

Reply:
Thank you guys for pointing this out. I will also handle the case the token is empty inside the middleware to early return